blogs/2018/09/07/blog-post.md

```diff
@@ -0,0 +1,72 @@
+---
+Order:
+TOCTitle: Using Azure Pipelines
+PageTitle: Improved Engineering with Azure Pipelines
+MetaDescription: Improved Engineering with Azure Pipelines
+MetaSocialImage: /assets/blogs/2018/09/05/github_pr_hero.png
+Date: 2018-09-07
+ShortDescription: Improved Engineering with Azure Pipelines
+Author: João Moreno
+---
+# Improved Engineering with Azure Pipelines
+
+September 7, 2018 João Moreno, [@joaomoreno](https://twitter.com/joaomoreno)
+
+One of my responsibilities as a developer on the Visual Studio Code team is to maintain and improve our build and CI infrastructure. Given the latest feature announcements from [Azure Pipelines](#TODO@PJ link to pipelines announcement post), the Visual Studio Code team has dramatically changed how we leverage Microsoft’s technologies to provide a better collaboration platform for both our developers as well as our users. In this blog post I’ll guide you through a bit of Visual Studio Code’s history, focusing on our CI processes and tools and how they have changed over time.
```

Comment:
Is it https://azure.microsoft.com/en-us/blog/announcing-azure-pipelines-with-unlimited-ci-cd-minutes-for-open-source/